app/controllers/application_controller.rb

```diff
@@ -22,6 +22,11 @@ def user_with_groups
 
   private
 
+  sig { returns(T::Boolean) }
+  def deposit?
+    params[:commit] == 'Deposit'
```

Comment:
I don't think this belongs in the Application controller.  Please move to a mixin or a subclass.

Reply:
Moved to a new controller that I subclass Collections and Works from to keep it separate from other controllers.  If we use a module, we'd have to pass in the params which is not as clean reading in the code.